ra2ce/analysis/losses/losses_base.py

```diff
@@ -442,7 +442,8 @@ def _calculate_production_loss_per_capita(
                 for duration, loss_ratio in zip(
                     duration_steps, functionality_loss_ratios
                 )
-            )
+            ) * sum(duration_steps)
```

Comment:
`vlh_trip_type_event = sum(
                (
                    intensity_trip_type
                    * occupancy_trip_type
                    * duration
                    * loss_ratio
                    * self.production_loss_per_capita_per_hour
                    * duration
                )
                / _divisor
                for duration, loss_ratio in zip(
                    duration_steps, functionality_loss_ratios
                )
            )`

Reply:
@MatthiasHauthDeltares Thanks.

Here I think this is the right formulation:

for each step of the resilience curve:

intensity x occupancy x loss_ratio x duration x productivity_loss_rate x duration


so for the whole resilience curve:

Sigma(intensity x occupancy x loss_ratio x duration x productivity_loss_rate x duration)